app/assets/javascripts/bank_transactions.es6

```diff
@@ -0,0 +1,13 @@
+$(document).ready(function(){
+    document.body.addEventListener('ajax:success', function(event) {

```

Comment:
This event will be triggered for all `'ajax:success'` events. I suggest we restrict to only the "remove transactions" events by adding a condition like so:
```javascript
document.body.addEventListener('ajax:success', function(event) {
  if (!event.srcElement.classList.contains('remote-remove-transaction')) return
```
same for the `'ajax:error'` event.

And add that class to the form generated by `button_to` in the view with the `form_class` attribute : 
```ruby
<%= button_to(
      t('.remove_link'),
      remove_transaction_type_citizens_bank_transaction_path(bank_transaction),
      method: :patch,
      class: 'button-as-link',
      form_class: 'remote-remove-transaction',
      remote: true
    ) %>
```

Reply:
Doh. Of course, I should have realised that. Thanks.